src/reduxForm.js

```diff
@@ -223,7 +223,10 @@ const createReduxForm =
               return promise
             }
             this.submitPromise = promise
-            return promise.then(this.submitCompleted, this.submitCompleted)
+            return promise.then(this.submitCompleted, err => {
```

Comment:
As reported in #1266, this line throws `undefined:1 Uncaught (in promise) Object { ...errors }` if there are validation errors.

If I do this
```
return promise.then(this.submitCompleted, err => {
  this.submitCompleted()
  return Promise.reject(err)
}).catch(err => {
  console.log('==> CATCH', err)
})
```
the error is caught. Let's continue discussion in #1266